piptools/cache.py

```diff
@@ -121,10 +121,6 @@ def __setitem__(self, ireq, values):
         self.cache[pkgname][pkgversion_and_extras] = values
         self.write_cache()
 
-    def get(self, ireq, default=None):
```

Comment:
NOTE: usage of this method has been removed here https://github.com/jazzband/pip-tools/pull/153/files#diff-4a3323f7294510dd8436d96838ca4a61L308